File: map.js
var results1 = [1, 2, 3, 4, 5, 6].map(function (x) {
  return x++;
});

var results2 = ["ca", "fa", "ma", "ra"].map(function (x) {
  return x + "t";
});

var results3 = [true, false].map(function (x) {
  return !x;
});

console.log(results1);
console.log(results2);
console.log(results3);

File: map.py
results1 = map(lambda x: x + 1, [1, 2, 3, 4, 5, 6]);

results2 = map(lambda x: x + "t", ["ca", "fa", "ma", "ra"]);

results3 = map(lambda x: not x, [True, False]);

print(list(results1))
print(list(results2))
print(list(results3))

